subsys/bluetooth/audio/shell/bap.c

```diff
@@ -1631,7 +1631,15 @@ static void conn_list_eps(struct bt_conn *conn, void *data)
 		const struct bt_bap_ep *ep = snks[conn_index][i];
 
 		if (ep != NULL) {
-			shell_print(sh, "    #%u: ep %p", i, ep);
+			struct bt_bap_ep_info ep_info;
+			int err;
+
+			err = bt_bap_ep_get_info(ep, &ep_info);
+			if (err == 0) {
+				shell_print(sh, "    #%u: ep %p (state: %d)", i, ep, ep_info.state);
+			} else {
+				shell_print(sh, "    #%u: ep %p (get ep info err: %d)", i, ep, err);
+			}
```

Comment:
I would probably just omit the error cases. 

If you want to keep them, we should use `shell_error` instead. 

The reason why I'd omit them, is that the function can only fail with invalid input, and we don't supply any invalid input here.

Reply:
Ok. I will omit the error case then